packages/manager/apps/pci-rancher/src/hooks/useRancher.tsx

```diff
@@ -13,11 +13,15 @@ export const useRancher = () => {
 
 export const ranchersQueryKey = (projectId: string) => ['project', projectId];
 
-export const useRanchers = () => {
+export const useRanchers = ({
+  shouldDisableRefetch = false,
```

Comment:
**question:** `shouldDisableRefetch` is not sent as true from anywhere no? Isn't it redundant?